modules/recommended-configurable-storage-technology.adoc

```diff
@@ -23,14 +23,15 @@ technologies for the given {product-title} cluster application.
 .Recommended and configurable storage technology
 [options="header"]
 |===
-|Storage type |ROX footnoteref:[rox,ReadOnlyMany]|RWX footnoteref:[rwx,ReadWriteMany] |Registry|Scaled registry|Metrics|Logging|Apps
+|Storage type |ROX footnoteref:[rox,ReadOnlyMany]|RWX footnoteref:[rwx,ReadWriteMany] |Registry|Scaled registry|Metrics footnoteref:[metrics-prometheus,Prometheus is the the underlying technology used for metrics.]|Logging|Apps
```

Comment:
@jboxman Do these footnotes appear properly in a preview? They do not render in GitHub.

Reply:
I don't know if GitHub understands footnotes. It does render locally successfully.